scratch3/blocks.js

```diff
@@ -553,8 +553,10 @@ class BlockView {
       innerWidth,
     )
 
-    // Center the label text inside small reporters.
-    if (this.isReporter) {
+    // Center the label text inside reporters with short names.
+    //
+    // Scratch doesn't do this for lists for some reason??
+    if (this.isReporter && this.info.category !== "list") {
```

Comment:
I've made this "is not list" rather than "is variable or parameter" because it will only matter in the case of a very short e.g. Motion reporter block, and I'm not aware of any -- I'll update this if we find one (in a non-English language maybe)?

Reply:
I don't know if I've made this clear or not but the (x) motion block is a real Scratch block, it's the (x position) block in Czech

Reply:
I'll update this.